popSimManu.tex

```diff
@@ -391,13 +412,14 @@ \subsection*{Use case: comparing methods of demographic inference}
 available from \url{https://github.com/popsim-consortium/analysis}.
 These workflows allow efficient computing in multicore or cluster environments.
 
+%ALG: When talking about the Snakemake workflow, consistently use the word “workflow”, not “pipeline”. 
```

Comment:
yup agreed

Reply:
done

Reply:
@petrelharp are you making these changes? Because I will push with the approved changes.